### DISCUSSION TITLE
Thank you for this one

### DISCUSSION THREAD

**Discussion by intandomayenziwe** [cyrilletuzi/vscode-angular-schematics#759] [General]
It makes everyday coding so much fun. I don't have to worry about messing up the hierarchy of components during development, I can easily place them where they are needed and are relevant.

@cyrilletuzi I'll buy you coffee someday soon :-)

**Comment by cyrilletuzi** [comment]
Hi @intandomayenziwe, thank you for taking time to say hello, I appreciate it a lot. 👍🏻 

I don't know if you mean a real coffee or if it's a metaphor (as we seem to live quite far away from each other), but be aware that there will be very soon a way to recompense my work while getting access to an even better tool. I'll keep you updated once it's ready.

Cheers.

**Comment by cyrilletuzi** [comment]
Hi @intandomayenziwe, for information, an extended version of the extension is now available: **Schematics Pro**. You will find all the details on the [dedicated website](https://www.cyrilletuzi.com/schematics-pro/).

So if you were serious about retribution, it is the occasion to do so while getting access to new features. :)
